docs/platform/howto/attach-vpc-aws-tgw.md

```diff
@@ -1,26 +1,28 @@
 ---
 title: Attach VPCs to an AWS Transit Gateway
+pro: true
 ---
 
-[AWS Transit Gateway (TGW)](https://aws.amazon.com/transit-gateway/)
-enables transitive routing from on-premises networks through VPN and
-from other VPC. By creating a Transit Gateway VPC attachment, services
-in an Aiven Project VPC can route traffic to all other networks
-attached - directly or indirectly - to the Transit Gateway.
+[AWS Transit Gateway (TGW)](https://aws.amazon.com/transit-gateway/) enables transitive routing from on-premises networks through VPN and from other VPC.
 
-## Set up a project VPC
+## How it works
 
-Create a VPC on the Aiven platform in the same region as your Transit
-Gateway.
+By creating a Transit Gateway VPC attachment, services in an Aiven Project VPC can route
+traffic to all other networks attached - directly or indirectly - to the Transit Gateway.
```

Comment:
Could we avoid using dashes? If they can be replaced by commas, let's do so. These dashes are not meant for this kind of use, but only for compound nouns.